### DISCUSSION TITLE
support integration with external apis like google search, wolfram alpha apis ?

### DISCUSSION THREAD

**Discussion by santosh0787** [open-webui/open-webui#1597] [Ideas]
Hello Team, 
is there any plan to support integration with external apis like google search, wolfram alpha, zapier apis similar to chatgpt-plus plugins?

**Comment by flefevre** [comment]
Hello,


there is already an integration with google search


https://docs.openwebui.com/tutorial/web_search
https://docs.openwebui.com/getting-started/env-configuration#web-search

**Comment by FarisZR** [comment]
is there anything planned to add Wolfram Alpha support? it's the killer Chatgpt feature for me.
